.azure-pipelines/integration-test.yml

```diff
@@ -0,0 +1,111 @@
+trigger:
+- main
+
+pr:
+  branches:
+    include:
+    - main
+  drafts: false
+
+jobs:
+- job: IntegrationTest
+  displayName: Integration test
+  strategy:
+    matrix:
+      cuda12:
+        containerImage: ghcr.io/microsoft/mscclpp/mscclpp:base-dev-cuda12.2
```

Comment:
Shall we leverage the latest docker which is 12.4

Reply:
Need to prepare MSCCL algorithm XMLs as well.

Reply:
Update in latest commit: Change the containerImage to cuda12.4. Add the msccl algorithm XMLs.